### PR TITLE
Adding initial model for the database

### PR DESCRIPTION
**Pull request by VeronaGH**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -32,6 +32,11 @@
             <artifactId>spring-boot-starter-data-mongodb</artifactId>
             <version>${spring-boot.version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.jetbrains</groupId>
+            <artifactId>annotations-java5</artifactId>
```
Comment (ikhmurchik) [inline pom.xml]: why you need this?

Comment (VeronaGH) [inline pom.xml]: I added some dependencies in order to run some examples. Now this dependency is not need, I suppose...

Comment (VeronaGH) [inline pom.xml]: Next commit will be free of their.

Comment (ikhmurchik) [inline pom.xml]: ok, i'll waiting

**Pull request merged**